neurovault/apps/statmaps/views.py

```diff
@@ -375,7 +375,7 @@ def upload_folder(request, collection_cid):
                             dom = minidom.parse(os.path.join(root, fname))
                             for atlas in dom.getElementsByTagName("summaryimagefile"):
                                 print "found atlas"
-                                path, base, ext = split_filename(atlas.lastChild.nodeValue)
+                                path, base = os.path.split(atlas.lastChild.nodeValue)
```

Comment:
the problem was for atlases whose base had a period in it. in those cases it would divide the part before and after the period and call them base and ext, but only base is used later so the stuff after the period was lost